src/pages/trackable.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState } from 'react'
+import { Box, Flex, Button, Image, Collapse, FormControl, FormLabel, Input, FormErrorMessage, Icon, Spinner, Text } from '@chakra-ui/core'
+import Header from '../components/header'
+import { useHistory } from 'react-router-dom';
+import debug from 'debug'
+import { Trackable } from '../generated/graphql'
+import { useForm } from 'react-hook-form';
+import { upload, getUrl } from '../lib/skynet';
+import { gql, useQuery, useMutation } from '@apollo/client';
+
+const log = debug("pages.trackable")
+
+type AddTrackableFormData = {
+    name: string
+    image: FileList
+}
+
+const CREATE_TRACKABLE = gql`
+    mutation CreateTrackable($input: CreateTrackableInput!) {
+        createTrackable(input: $input) {
+            collection {
+                did
+            }
+            trackable {
+                did
+                name
+                image
+            }
+        }
+    }
+`
+
+const GET_TRACKABLES = gql`
+    {
+        me {
```

Comment:
is the *get* user specific now?

Reply:
Not sure. I just copied the old `index.tsx` to here to make room for a new `index.tsx`. I didn't change nor use (yet) any of this code, but thought it might still be needed by others so didn't want to just delete it.